### PR TITLE
Prevent tridents from being picked up

### LINKED ISSUE #3039: /fireball trident - Allows players to get free tridents
Version:
```
27.02 13:34:32 [Server] Server thread/INFO Server version: 1.15.1-R0.1-SNAPSHOT git-Spigot-037559e-5c6b0dc (MC: 1.15.1)
27.02 13:34:32 [Server] Server thread/INFO EssentialsX version: 2.17.2.0
27.02 13:34:32 [Server] Server thread/INFO LuckPerms version: 5.0.39
27.02 13:34:32 [Server] Server thread/INFO PlaceholderAPI version: 2.9.2
27.02 13:34:32 [Server] Server thread/INFO Vault version: 1.7.2-b107
27.02 13:34:32 [Server] Server thread/INFO ChestShop version: 3.10-SNAPSHOT (build 149)
27.02 13:34:32 [Server] Server thread/INFO Citizens version: 2.0.26-SNAPSHOT (build 1790)
27.02 13:34:32 [Server] Server thread/INFO EssentialsXChat version: 2.17.2.0
27.02 13:34:32 [Server] Server thread/INFO EssentialsXSpawn version: 2.17.2.0
27.02 13:34:32 [Server] Server thread/INFO You are running an unsupported server version!
```

**Description**  
When someone runs /fireball trident a player can pick up the trident thrown.

**Steps to reproduce**  
/fireball trident

**Expected behavior**  
Should not allow a player to pick up a trident like one thrown by a mob.

### PR DESCRIPTION
**Pull request by JRoy**
Fixes #3039 *properly*

### CODE REVIEW

**Review by pop4959 — DISMISSED** [review]

File: Essentials/src/com/earth2me/essentials/EssentialsPlayerListener.java [hunk 1]
```diff
@@ -68,9 +69,22 @@ private static boolean isCommandSendEvent() {
         }
     }
 
+    private static boolean isArrowPickupEvent() {
+        try {
+            Class.forName("org.bukkit.event.player.PlayerPickupArrowEvent");
```
Comment (mdcfe) [inline Essentials/src/com/earth2me/essentials/EssentialsPlayerListener.java:74]: When was this event added?

Comment (JRoy) [reply Essentials/src/com/earth2me/essentials/EssentialsPlayerListener.java:74]: In the middle of the 1.9 or 1.10 lifecycle

Tridents were added in 1.13 so we don't need a backport for 1.8

Comment (pop4959) [reply Essentials/src/com/earth2me/essentials/EssentialsPlayerListener.java:74]: @md678685 Looking through various API revisions, it looks like this dates as far back as at least 1.9.4 ([source](https://helpch.at/docs/1.9.4/org/bukkit/event/player/PlayerPickupArrowEvent.html)).

Comment (mdcfe) [reply Essentials/src/com/earth2me/essentials/EssentialsPlayerListener.java:74]: Is there an equivalent for 1.8.8 to limit picking up arrows? If not, this isn't a blocker.

Comment (JRoy) [reply Essentials/src/com/earth2me/essentials/EssentialsPlayerListener.java:74]: I added 1.8.8 support, tested and works.

**Review by mdcfe — APPROVED** [review]

**Review by pop4959 — APPROVED** [review]

**Pull request merged**